manifests/trust-manager-ca-bundle.yaml

```diff
@@ -7,12 +7,11 @@ apiVersion: trust.cert-manager.io/v1alpha1
 kind: Bundle
 metadata:
   name: linkerd-identity-trust-roots
-  namespace: linkerd
 spec:
   sources:
   - secret:
       name: "linkerd-identity-trust-roots"
-      key: "ca.crt"
+      key: "tls.crt"
```

Comment:
It is the signed `tls.crt` which should be copied to the ca-bundle. It doesn't matter when it is self-signed, but when it's a vault signed intermediate, the ca.crt will be the root certificate from vault.